src/test/java/com/jwplayer/southpaw/util/ByteArraySetTest.java

```diff
@@ -46,6 +49,1682 @@ public ByteArraySet createSmallSet() {
+        stringVals.add("1ldkKt");
+        stringVals.add("NjhunK");
+        stringVals.add("1m09Mj");
+        stringVals.add("pS1QYi");
+        stringVals.add("IzZqui");
+        stringVals.add("Q2YY4S");
+        stringVals.add("2ahZ4l");
+        stringVals.add("gaJGG5");
+        stringVals.add("xj3blp");
+        stringVals.add("xMhGUA");
+        stringVals.add("1MlUVQ");
+        stringVals.add("g1NDDq");
+        stringVals.add("EY6vXO");
+        stringVals.add("9cTQSZ");
+        stringVals.add("MWuj4w");
+        stringVals.add("hasr7r");
+        stringVals.add("nWqpj5");
+        stringVals.add("MgBiXt");
+        stringVals.add("t65T7v");
+        stringVals.add("wHlIco");
+        stringVals.add("hRAHav");
+        stringVals.add("If0jLA");
+        stringVals.add("pBel9o");
+        stringVals.add("tXxNdk");
+        stringVals.add("uWRdwz");
+        stringVals.add("Lbv3b8");
+        stringVals.add("rW0V4L");
+        stringVals.add("rKhcGH");
+        stringVals.add("ztWe8U");
+        stringVals.add("zLTNNO");
+        stringVals.add("Ola90f");
+        stringVals.add("ohOqgT");
+        stringVals.add("mbZoS0");
+        stringVals.add("PcDCiI");
+        stringVals.add("56vH6o");
+        stringVals.add("XMPBK7");
+        stringVals.add("K5RNsR");
+        stringVals.add("k9AZ4y");
+        stringVals.add("JxnerK");
+        stringVals.add("zFAtxy");
+        stringVals.add("lfB8RK");
+        stringVals.add("9sQ2gq");
+        stringVals.add("cPaxfM");
+        stringVals.add("rXX054");
+        stringVals.add("wWYLN8");
+        stringVals.add("Ni4st9");
+        stringVals.add("hG3lCA");
+        stringVals.add("rWTIjz");
+        stringVals.add("P2B01G");
+        stringVals.add("PnBv3g");
+        stringVals.add("W7BDeN");
+        stringVals.add("ipnNjI");
+        stringVals.add("gF1LVD");
+        stringVals.add("EWm8nh");
+        stringVals.add("mIL0ef");
+        stringVals.add("opuSW5");
+        stringVals.add("BCUw88");
+        stringVals.add("cn4dKT");
+        stringVals.add("CNYAsx");
+        stringVals.add("64aymk");
+        stringVals.add("0MpN1R");
+        stringVals.add("il8dgq");
+        stringVals.add("TQKRtN");
+        stringVals.add("zmt8p1");
+        stringVals.add("NB2ztN");
+        stringVals.add("b4BK74");
+        stringVals.add("t3uNny");
+        stringVals.add("2TL7lC");
+        stringVals.add("xbQxTL");
+        stringVals.add("m1QBnY");
+        stringVals.add("H33uUn");
+        stringVals.add("YYIOMC");
+        stringVals.add("Uo16hH");
+        stringVals.add("Eli8du");
+        stringVals.add("PY3nmS");
+        stringVals.add("PYLQRm");
+        stringVals.add("eVlgM0");
+        stringVals.add("zl99JF");
+        stringVals.add("QIBt20");
+        stringVals.add("OaaY6R");
+        stringVals.add("G8U76E");
+        stringVals.add("Ir06pg");
+        stringVals.add("DVT2j5");
+
+        List<String> deletedVals = new ArrayList<String>();
```

Comment:
If this is a subset of the stringVals we could just take a random X number of items from `stringVals` to construct this or generate this like suggested above

Reply:
Not really as the bug will only appear when we delete certain values.